Lib/asyncio/selector_events.py

```diff
@@ -540,6 +540,22 @@ def _sock_accept(self, fut, registered, sock):
         else:
             fut.set_result((conn, address))
 
+    async def _sendfile_native(self, transp, file, offset, count):
+        del self._transports[transp._sock_fd]
+        resume_reading = transp.is_reading()
+        transp.pause_reading()
+        fut = transp._make_empty_waiter()
+        await fut
+        sock = transp._sock
```

Comment:
Do you need the `sock` variable? Just refer to `transp._sock` directly in `self.sock_sendfile` call.